README.md

```diff
@@ -1,45 +1,58 @@
-# KCL
+<h1 align="center">KCLVM</h1>
 
-![license](https://img.shields.io/badge/license-Apache--2.0-green.svg)
-[![Continuous Integration](https://github.com/KusionStack/KCLVM/actions/workflows/github-actions.yaml/badge.svg)](https://github.com/KusionStack/KCLVM/actions?query=branch%3Amain)
+<p align="center">
+<a href="./README.md">English</a> | <a href="./README-zh.md">简体中文</a>
+</p>
+<p align="center">
+<a href="#introduction">Introduction</a> | <a href="#features">Features</a> | <a href="#features">Features</a> | <a href="#installation">Installation</a> | <a href="#what-is-it-for">What is it for</a> | <a href="#documentation">Documentation</a> | <a href="#contributing">Contributing</a> | <a href="#roadmap">Roadmap</a> | <a href="#license">License</a>
+</p>
 
-[中文](./README_ZH.md)
+<p align="center">
+	<img src="https://img.shields.io/github/release/KusionStack/KCLVM.svg">
+  <img src="https://img.shields.io/badge/Rust-1.60-blue.svg">
+  <img src="https://coveralls.io/repos/github/KusionStack/KCLVM/badge.svg">
+  <img src="https://img.shields.io/github/license/KusionStack/KCLVM.svg">
+  <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square">
+</p>
 
-Kusion Configuration Language (KCL) is an open source configuration language mainly used in [Kusion Stack](https://kusionstack.io). KCL is a statically typed language for configuration and policy scenarios, based on concepts such as declarative and Object-Oriented Programming (OOP) paradigms.
+<p align="center">
+	<img src="https://github.com/KusionStack/KCLVM/actions/workflows/ubuntu_test.yaml/badge.svg">
+  <img src="https://github.com/KusionStack/KCLVM/actions/workflows/macos_test.yaml/badge.svg">
+  <img src="https://github.com/KusionStack/KCLVM/actions/workflows/windows_test.yaml/badge.svg">
```

Comment:
simplify action name, for example: rename "build-and-test-macos" to "macos"

Reply:
only include one named "KCL" action badge, ant move to the first.

Reply:
> only include one named "KCL" action badge, ant move to the first.

fixed.